ee/branding/pages/index.tsx

```diff
@@ -1,158 +1,28 @@
-        <form onSubmit={onSubmit}>
-          <div className='flex flex-col space-y-2'>
-            <div className='form-control w-full md:w-1/2'>
-              <label className='label'>
-                <span className='label-text'>{t('bui-shared-logo-url')}</span>
-              </label>
-              <input
-                type='url'
-                id='logoUrl'
-                className='input-bordered input'
-                onChange={onChange}
-                value={branding.logoUrl || ''}
-                placeholder='https://company.com/logo.png'
-              />
-              <label className='label'>
-                <span className='label-text-alt'>{t('bui-shared-logo-url-desc')}</span>
-              </label>
-            </div>
-            <div className='form-control w-full md:w-1/2'>
-              <label className='label'>
-                <span className='label-text'>{t('bui-shared-favicon-url')}</span>
-              </label>
-              <input
-                type='url'
-                id='faviconUrl'
-                className='input-bordered input'
-                onChange={onChange}
-                value={branding.faviconUrl || ''}
-                placeholder='https://company.com/favicon.ico'
-              />
-              <label className='label'>
-                <span className='label-text-alt'>{t('bui-shared-favicon-url-desc')}</span>
-              </label>
-            </div>
-            <div className='form-control w-full md:w-1/2'>
-              <label className='label'>
-                <span className='label-text'>{t('branding_company_name_label')}</span>
-              </label>
-              <input
-                type='text'
-                id='companyName'
-                className='input-bordered input'
-                onChange={onChange}
-                value={branding.companyName || ''}
-                placeholder={t('branding_company_name_label')}
-              />
-              <label className='label'>
-                <span className='label-text-alt'>{t('branding_company_name_alt')}</span>
-              </label>
-            </div>
-            <div className='form-control'>
-              <label className='label'>
-                <span className='label-text'>{t('bui-shared-primary-color')}</span>
-              </label>
-              <input
-                type='color'
-                id='primaryColor'
-                onChange={onChange}
-                value={branding.primaryColor || '#25c2a0'}
-              />
-              <label className='label'>
-                <span className='label-text-alt'>{t('bui-shared-primary-color-desc')}</span>
-              </label>
-            </div>
-            <div className='mt-5'>
-              <ButtonPrimary loading={loading}>{t('bui-shared-save-changes')}</ButtonPrimary>
-            </div>
-          </div>
-        </form>
-      </div>
-    </>
+    <BrandingForm
+      defaults={{ primaryColor: '#25c2a0' }}
```

Comment:
It's helpful to have either define const for primaryColor or create one variation in tailwind config file.

Reply:
@deepakprabhakara Should we expose [these](https://github.com/boxyhq/jackson/blob/2c80f7c9201947e1f7ccfaebf8ce96dfe73d58c7/ee/branding/utils.ts#L5) values as default in the form ?

Reply:
Sounds good, just double check the actual defaults match those

Reply:
I think its best to display the default for the admin portal branding settings while leaving it empty for the other branding forms (Identity Federation and Product branding form).

Reply:
For the other branding forms if default is actually empty then it makes sense, otherwise the defaults should be used.